### PR TITLE
generic: get rid of outdated &hasi; entity

### PR DESCRIPTION
**Pull request by taroth21**
### PR creator: Description

To correct the productnames to the approved ones as per our terminology database several steps are needed. This is the first step: 

The following preparations were needed before we can fix the value of `&productname;` via doc-kit:

  * replace `[T,t]he &hasi;` with `&productname;` where appropriate
  * replace `[T,t]he &hasi;` with `&ha; (software)` where appropriate

### PR creator: Are there any relevant issues/feature requests?

* jsc#DOCTEAM-1133

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP5
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
  - [x] 15 SP1
- SLE-HA 12
  - [x] 12 SP5
  - [x] 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by taroth21** [comment]
@tahliar : Could you please have a look and backport when approved and merged? For Code 12, the concept of modules & extensions did not exist yet in the way it was defined for Code 15 (and the 'Modules & Extensions Quick Start' did not exist. Therefore for 12 SP5 and SP4, you need to adjust the para/section in ha_concepts.xml, L. 53/54 accordingly. The rest of the changes should apply to both Code 15 and Code 12 versions.

File: xml/ha_concepts.xml [hunk 1]
```diff
@@ -47,10 +47,11 @@
       </dm:docmanager>
     </info>
     <sect1 xml:id="sec-ha-availability">
-  <title>Availability as extension</title>
+  <title>Availability as module or extension</title>
```
Comment (tahliar) [inline xml/ha_concepts.xml:50]:
```suggestion
  <title>Availability as a module or extension</title>
```

File: xml/ha_concepts.xml [hunk 2]
```diff
@@ -47,10 +47,11 @@
...
 
   <para>
-   The &hasi; is available as an extension to &sls; &productnumber;.
+   &ha; is available as module or extension for several products. For details, see
```
Comment (tahliar) [inline xml/ha_concepts.xml:53]:
```suggestion
   &ha; is available as a module or extension for several products. For details, see
```

**Review by tahliar — COMMENTED** [review]
LGTM, thanks! Just two little nitpicks:

**Comment by taroth21** [comment]
@tahliar : Thanks, I have applied them now.

**Pull request merged**